### DISCUSSION TITLE
How to Transport the Function "CombineXpOrbs" to a Later Version of Carpet ?

### DISCUSSION THREAD

**Discussion by Arthur-Montgomery** [gnembon/fabric-carpet#1714] [Q&A]
How do I transport a former function to a later version of fabric carpet?
For example, what should I do if I want to have 'CombineXpOrbs' (which has already been removed since version 1.4.56) in, say, carpet for 1.19.2 mc?

**Comment by altrisi** [comment]
It's now a vanilla feature, the game merges orbs of the same size.

**Reply by Arthur-Montgomery** [reply]
Thank you! But when did they add this feature in vanilla mc? Since 1.18?
And I also want to use this feature in carpet 1.4.57, for mc 1.17.
Looks like I still need to figure out how to transport this to 1.4.57, or transport carpetCommandPermissionLevel to 1.4.55.
